web_src/css/home.css

```diff
@@ -67,13 +67,13 @@
   justify-content: center;
 }
 
-.page-footer .right-links > a {
+.page-footer .right-links > * + * {
```

Comment:
What is this selector?
That does not look like it should be used at all…

Reply:
Any element followed by another element, it's used to insert the `|` and spacing between items. This is a case where wildcard selector is useful.